RadarSDK/Radar.m

```diff
@@ -71,6 +71,12 @@ + (void)initializeWithPublishableKey:(NSString *)publishableKey {
     
 }
 
++ (void)initializeWithPublishableKey:(NSString *)publishableKey
+                              region:(RadarHostRegion)region {
+    [RadarSettings setHostRegion:region];
+    [Radar initializeWithPublishableKey:publishableKey];
+}
```

Comment:
If we use this way of setting the host (region) in Waypoint, it'll break the way we change the host in SU... that's probably alright (we just won't use this in Waypoint)

Reply:
Actually, we can set the `region` in demo app config, and use that to use this if set, otherwise, just using the original one. That will allow us to dogfood this.

Let's wait for the europe / na regions to go live and have this in Waypoint before we merge